Gooey.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name                  = 'Gooey'
-  s.version               = '1.0.1'
+  s.version               = '1.0.2'
```

Comment:
Does it make sense for this to be a podspec version bump? This only affects Carthage and testing which aren't in the scope of the pod.

Reply:
Good point!